.test-infra/dataproc/init-actions/flink.sh

```diff
@@ -155,7 +155,9 @@ function configure_flink() {
 # Settings applied by Cloud Dataproc initialization action
 jobmanager.rpc.address: ${master_hostname}
 jobmanager.heap.mb: ${flink_jobmanager_memory}
-taskmanager.heap.mb: ${flink_taskmanager_memory}
+taskmanager.memory.process.size: "${flink_taskmanager_memory} mb"
+taskmanager.memory.jvm-metaspace.size: 512 mb
+taskmanager.memory.managed.fraction: 0.5
```

Comment:
Should we set a fixed amount of memory here? 0.5 could be a lot.

Reply:
If you needed 0.5, then that's fine.

Reply:
Yes, the jobs started working after setting the fraction to 0.5. The default value is 0.4, so the difference might not be that huge.

Reply:
Sounds good!